lit_nlp/components/metrics.py

```diff
@@ -30,6 +30,7 @@
 from scipy import stats as scipy_stats
 from scipy.spatial import distance as scipy_distance
 from sklearn import metrics as sklearn_metrics
+import math
```

Comment:
only need to import math if you want to use math.nan   
if using np.nan, as you are below, you don't need this import